commands/show-bl-ip.go

```diff
@@ -50,35 +51,30 @@ func init() {
 	bootstrapCommands(blSourceIPs, blDestIPs)
 }
 
-func parseBLArgs(c *cli.Context) (string, string, bool, bool, error) {
+func parseBLArgs(c *cli.Context) (string, string, bool, bool, bool, error) {
 	db := c.Args().Get(0)
 	sort := c.String("sort")
 	connected := c.Bool("connected")
 	human := c.Bool("human-readable")
+	showNetNames := c.Bool("network-names")
 	if db == "" {
-		return db, sort, connected, human, cli.NewExitError("Specify a database", -1)
+		return db, sort, connected, human, showNetNames, cli.NewExitError("Specify a database", -1)
```

Comment:
Just a very minor suggestion but can we maybe consider having an errVal (or whatever you want to call it) in this particular case rather than the multiple returns? Just since we have so many params in each return. Will make future changes easier by only needing to modify the params in one place rather than three. So like:

`var errVal error = nil

if db == "" { 
   errVal = cli.NewExitError("Specify a database", -1)
} else if sort != "conn_count" && sort != "total_bytes" {
   errVal = cli.NewExitError("Invalid option passed to sort flag", -1)
}

return db, sort, connected, human, errVal`
}

It's not a big deal but just a suggestion

Reply:
Changed. :+1: